Versionize/Git/RepositoryExtensions.cs

```diff
@@ -9,9 +9,11 @@ namespace Versionize.Git;
 
 public static class RepositoryExtensions
 {
-    public static Tag? SelectVersionTag(this Repository repository, SemanticVersion version)
+    public static Tag? SelectVersionTag(this Repository repository, SemanticVersion version, bool omitTagVersionPrefix = false)
```

Comment:
@cabauman: This method is only used within the tests. Since it belongs to the public API, I didn't dare to remove it. However, you may want to keep that in mind and remove it at a later point in time, except if you know other projects that sit on top of versionize and use this API.